pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -2521,17 +2545,16 @@ status:
 		t.Errorf("Expected PipelineRun to be timed out, but condition reason is %s", reconciledRun.Status.GetCondition(apis.ConditionSucceeded))
 	}
 
-	// Check that the expected TaskRun was created
-	actual := getTaskRunCreations(t, clients.Pipeline.Actions(), 2)[0]
-
-	// The TaskRun timeout should be less than or equal to the PipelineRun timeout.
-	if actual.Spec.Timeout.Duration > prs[0].Spec.Timeout.Duration {
-		t.Errorf("TaskRun timeout %s should be less than or equal to PipelineRun timeout %s", actual.Spec.Timeout.Duration.String(), prs[0].Spec.Timeout.Duration.String())
+	// Check that there is a skipped task for the expected reason
+	if len(reconciledRun.Status.SkippedTasks) != 1 {
+		t.Errorf("expected one skipped task, found %d", len(reconciledRun.Status.SkippedTasks))
+	} else if reconciledRun.Status.SkippedTasks[0].Reason != v1beta1.PipelineTimedOutSkip {
+		t.Errorf("expected skipped reason to be '%s', but was '%s", v1beta1.PipelineTimedOutSkip, reconciledRun.Status.SkippedTasks[0].Reason)
```

Comment:
could you please update the description of the test to explain that it verifies that no new runs are created, and that the pipeline task is marked as skipped?

Reply:
👍